runtime/onert/core/src/compiler/LoweredGraph.cc

```diff
@@ -459,7 +459,7 @@ void LoweredGraph::dumpLowerInfo()
   {
     if (!e.second.empty())
     {
-      VERBOSE(Lower) << e.second;
+      VERBOSE(Lower) << e.second << std::endl;
```

Comment:
Nit: actually the whole if statement here was inserted by accident. My bad 😢

Reply:
I think it's not your mistake. I need to revert this change. After adding this, log had extra line after `Use Backends` item...
```
    - Def Backends    : { cpu(NHWC) }
    - Use Backends    : { cpu(NHWC) }

[Lower] Operand #7 LowerInfo
```